spras/containers.py

```diff
@@ -240,14 +242,98 @@ def run_container_docker(container: str, command: List[str], volumes: List[Tuple
     return out
 
 
-def run_container_singularity(container: str, command: List[str], volumes: List[Tuple[PurePath, PurePath]], working_dir: str, environment: str = 'SPRAS=True'):
+def create_cgroup() -> str:
```

Comment:
[I'm generally made uncomfortable with including such a specific function in this ever-growing file, and I would like to see profiling in general moved to a separate file (especially if we decided to add profiling for other platforms/container frameworks). Understandably, though, that is a little much.]